### PR TITLE
fix: Collectible screen design issues

### LINKED ISSUE #19701: Collectible page small design issues
# Bug Report

- [x] `Cross icon` grey background
- [x] `Three dots menu` grey background
- [x] `Account` and `Network` tabs grey background
- [ ]  No `Jump to button`: https://github.com/status-im/status-mobile/issues/19722
- [x]  Incorrect placement of `Photo is saved to your device` banner on Android only

Screenshot 2024-04-18 at 09 21 16
Screenshot 2024-04-18 at 09 24 04

Design is [here](https://www.figma.com/file/HKncH4wnDwZDAhc4AryK8Y/Wallet-for-Mobile?type=design&node-id=11571%3A111653&mode=design&t=bFidNev5bTnosMlN-1)

### Additional Information

- Status version: nightly 
- Operating System: Android, iOS

[comment]: # (Please, add logs/notes if necessary)

### PR DESCRIPTION
**Pull request by OmarBasem**
fixes: https://github.com/status-im/status-mobile/issues/19701

- Top bar icons background
- Account and network data item background color
- Photo saved toast position
- Toast icon not appearing
- Margin of image in lightbox

Screenshot_20240402_141910_Status

Screenshot_20240402_141910_Status

### CODE REVIEW

File: src/status_im/common/scroll_page/view.cljs [hunk 1]
```diff
@@ -64,9 +64,7 @@
         [quo/page-nav
          (cond-> {:margin-top     (safe-area/get-top)
                   :type           :no-title
-                  :background     (if (= 1 (reanimated/get-shared-value opacity-animation))
-                                    :blur
-                                    :photo)
+                  :background     :blur
```
Comment (OmarBasem) [inline src/status_im/common/scroll_page/view.cljs]: I was trying to look in the designs if there is any instance where that scrollable page component uses top button with type `photo` but couldn't find any. cc @J-Son89

File: src/status_im/contexts/wallet/collectible/options/view.cljs [hunk 2]
```diff
@@ -30,7 +30,6 @@
                                  #(rf/dispatch [:toasts/upsert
                                                 {:id              :random-id
                                                  :type            :positive
-                                                 :container-style {:bottom (when platform/android? 20)}
```
Comment (OmarBasem) [inline src/status_im/contexts/wallet/collectible/options/view.cljs:33]: It seems to me `:bottom (when platform/android? 20)` is not needed. cc @vkjr @J-Son89

**Review by ibrkhalil — APPROVED** [review]

File: src/status_im/common/scroll_page/view.cljs (hunk 1 above)
Comment (J-Son89) [reply src/status_im/common/scroll_page/view.cljs]: ah, I believe it's like this when the animation is complete so you might have to check that on the communitys page for example.

File: src/status_im/contexts/wallet/collectible/options/view.cljs (hunk 2 above)
Comment (J-Son89) [reply src/status_im/contexts/wallet/collectible/options/view.cljs:33]: I'm not sure about this one.

File: src/status_im/common/scroll_page/view.cljs (hunk 1 above)
Comment (J-Son89) [reply src/status_im/common/scroll_page/view.cljs]: e.g https://www.figma.com/file/h9wo4GipgZURbqqr1vShFN/Communities-for-Mobile?type=design&node-id=1733-127837&mode=design&t=IMyYUqCrxBlSuvzE-4

**Review by J-Son89 — APPROVED** [review]
looks good! one minor detail to check in  the community scroll page detail (if that component is still used there)

**Comment by churik** [comment]
Hello!
The e2e results are reviewed, the failed tests are not related to the PR.
PR can proceed to the next stage - I believe in this particular case `design review` will be enough.

**Comment by churik** [comment]
@Francesca-G please, review when you have a time! thanks.

**Review by Francesca-G — COMMENTED** [review]
Review comments  [here](https://www.figma.com/file/FuHW0rmIa5fQr5yyt3pXM6/Review-Wallet-%2F-Mobile?type=design&node-id=49%3A29095&mode=design&t=zC1DkSGqhJHQ5yPQ-1) 

I didn't add a comment about the missing Jump To button as it's already logged

**Review by Francesca-G — APPROVED** [review]

**Pull request merged**